src/containers/DataDisplay/DataViewDetail.tsx

```diff
@@ -14,9 +14,9 @@ const DataViewDetail: React.FC = () => {
   const { strings } = useLanguageProvider()
   const {
     push,
-    params: { tableName, id },
+    params: { dataViewCode, id },
```

Comment:
Since we now query a specific data view, we call by code rather than table. This just changes the variable name, but actually makes no difference to functionality as far as front-end goes.